Makefile

```diff
@@ -24,7 +24,7 @@ APPNAME = XRP
 APP_LOAD_PARAMS=--appFlags 0xa40 --path "44'/144'" --curve secp256k1 --curve ed25519 $(COMMON_LOAD_PARAMS)
```

Comment:
Makefile shoud be align with the boilerplate one.
Especially the link & compiler options.

Reply:
Updated.